### ISSUE TITLE
htop orange pi 

### ISSUE THREAD

**Issue by renewedbullet** [htop-dev/htop#1363]
Does htop 3.X not include kernel and OS version and amount of users logged in like 2.2 did?

![image](https://github.com/htop-dev/htop/assets/140523069/ab7fd53b-15e7-4fb3-b303-63a35290b189)

**Comment by renewedbullet** [comment]
similiar to this, 
![image](https://github.com/htop-dev/htop/assets/140523069/acd0653f-8a32-4030-a744-c133ce862d6e)

**Comment by BenBE** [comment]
There's a Meter for the System in 3.3, which can be seen in [this screenshot](https://social.chaotikum.org/deck/@benbe/111303264986839094):
![image](https://github.com/htop-dev/htop/assets/111196/71097f0c-bb92-4ba7-b46d-788987b360fe)

The number of logged in users is not currently available as a meter in standard htop; this is a modification of some derivative versions; if you can look for those changes and point them here, we might be able to integrate them.

**Comment by renewedbullet** [comment]
I apologize for being dumb, is this the files in which your looking for? 
[Downloads.zip](https://github.com/htop-dev/htop/files/13873931/Downloads.zip)

**Comment by renewedbullet** [comment]
OS_Meter.c OS_meter.h kernel_Meter.c Kernel_meter.h

**Comment by BenBE** [comment]
Yes, do you have a repository you can point to for these files?

I noticed they might likely be from:
https://github.com/avafinger/htop_2.2.2/

Given there's [some history with that fork of htop](https://github.com/htop-dev/htop/issues/523#issuecomment-821986130), I'm quite reluctant to import patches from there directly (also that fork is based on old htop 2.2). That being said, all the mentioned meters are easy enough to re-implement from scratch.

**Comment by renewedbullet** [comment]
Yes it is, i wasn't aware of any conflicts I apologize just native htop  i installed was missing those figured it would be a nice QoL upgrade then I saw them on 2.2 was curious why an older version had it. If its to much no worries.

**Comment by BenBE** [comment]
Writing those meters is not too difficult. For the OS/Kernel/Distro meters we already have the code to gather the information, so that's basically some boilerplate to implement.

For the logged-in user count the function to get that information is not to difficult either; it's just a matter of implementing it in the platform layer, as that meter is relevant for more than just Linux.
